### DISCUSSION TITLE
Welcome! Introduce yourself here

### DISCUSSION THREAD

**Discussion by thibaudcolas** [django-recaptcha/django-recaptcha#331] [General]
👋 Hi all! Please use this discussion thread to introduce yourselves, so we get to know a bit about everyone using, contributing, or maintaining django-recaptcha :)

We run this package as an independent community of maintainers, it’s nice to know who’s here, doing what, what projects you use this on, etc.

It’s up to you what to mention in your introduction but we suggest things like your name, pronouns, location, a quick intro about you, hobbies if you feel like it, and what projects you use this for.

**Comment by thibaudcolas** [comment]
And I get to start! I’m Thibaud, he/him, based in Europe. Open source contributor for about 10 years now, I work at [@torchbox](https://github.com/torchbox), primarily on [@wagtail](https://github.com/wagtail/wagtail). In my free time I also contribute to [@django](https://github.com/django/django) and other things.

I’ve used reCAPTCHA in the past but I don’t think I’ve actually ever used `django-recaptcha` myself. I found myself helping with maintenance because my employer uses the package on projects, and we have good relationships with people at [@praekelt](https://github.com/praekelt) who were previously maintaining this. So I offered to take over, and here we are.

Though I don’t use this project directly, I have plenty of experience maintaining packages for personal projects, Torchbox, Wagtail, and elsewhere. In particular I’m involved with maintenance of everything under [@wagtail](https://github.com/wagtail) and [@wagtail-nest](https://github.com/wagtail-nest) (our equivalent to [@jazzband](https://github.com/jazzband)). I’ve always liked the collaborative aspects of open source, so would love to see this project keep on working for people with a model that reduces the bus factor and encourages more contributions.

Edit: I actually have some involvement with [django/djangoproject.com](https://github.com/django/djangoproject.com) as a contributor, which does use this package :) And I now realise [DjangoGirls/djangogirls](https://github.com/DjangoGirls/djangogirls) uses this too, I have a few commits there as well.

**Comment by Stormheg** [comment]
Second, hi! 👋

I'm Storm, he/him and based in the Netherlands. I work as a freelancer for various clients using Django/Wagtail/React. I'm a member of Wagtails core team, Wagtail Nest and a member of the Django Software Foundation. I've worked with @thibaudcolas and like to think I know him quite well 😄 

I occasionally use this package on some client sites, keeping it in good health is important to me.

In my day job I use a lot of open source for free. Contributing back to the amazing open source community by volunteering as a maintainer is the least I can do to give back and make it a fair exchange. You can find me on [Wagtails Slack](https://github.com/wagtail/wagtail/wiki/Slack) or [Django's Discord](https://www.djangoproject.com/community/), though I'm not a very active Discord user.

**Comment by tbrlpld** [comment]
Also hello from me 👋 I'm Tibor and based in California. I am a long term contractor with @torchbox, which is where I got to know @thibaudcolas and am working on projects with @django and @wagtail. 

I have collaborated with @thibaudcolas and @Stormheg before, when we worked on the [custom Sphinx theme ](https://github.com/wagtail/sphinx-wagtail-theme) for the [Wagtail docs](https://docs.wagtail.org/en/stable/).

I work on several projects that use `django-recaptcha` and got involved, when upgrading Django was blocked by this package, to make sure it stays compatible with the latest Django releases.

**Comment by Andrew-Chen-Wang** [comment]
Hi; I'm based in NYC. I work in startup land and share my learnings along the way. I _try_ to maintain packages that once became inactive like in @jazzband, though I've started to lose a lil bit of time over the years of startups 😅 . I use `django-recaptcha` frequently for small projects' authentication. I try to get startups to contribute back to OSS:)

**Reply by thibaudcolas** [reply]
Thanks for giving those intros a go @Andrew-Chen-Wang! Great to have you here

**Comment by rgs258** [comment]
Hi,

I'm Ryan, he/him, based in Philadelphia PA where I work for @wharton on our research platform, WRDS. We use reCAPTCHA, via this package, for the usual reasons on most forms outside our paywall. I've contributed some PRs here or there, but haven't before been regarded as a "maintainer" of any open source projects. I'm happy to help out and keep this project active, but I'm going to be learning and looking for some hands to hold as I do.

Thank you @thibaudcolas for setting this new course. Honestly, if it weren't for you and Matt W, I'd probably see go home later most nights - thank you for all your contributions to Wagtail and Django!

**Reply by thibaudcolas** [reply]
@rgs258 welcome to the "maintainer" club 😄 we’re all figuring it out as we go along
